load_test.py

```diff
@@ -0,0 +1,74 @@
+import lumigator_demo as ld
+import pandas as pd
+import multiprocessing
+from loguru import logger
+from functools import partial
+
+# Global Vars
+
+TEAM_NAME = "stress_testers"
+DATASET_NAME = "thunderbird.csv"
+DATASET_ID = "db7ff8c2-a255-4d75-915d-77ba73affc53" # thunderbird dataset 
```

Comment:
At least for now, the dataset ids should be two: one for the CSV file, one for the HF datasets directory. Something like

```suggestion
DATASET_ID = "db7ff8c2-a255-4d75-915d-77ba73affc53" # thunderbird dataset
DATASET_ID_HF = "f5d54efa-247d-4910-9393-f6003da9fb68" # thunderbird dataset in HF format
```